tools/wptrunner/requirements_firefox.txt

```diff
@@ -4,3 +4,5 @@ mozprocess == 0.26
 mozcrash == 1.0
 mozrunner == 6.14
 mozleak == 0.1
+mozinstall == 1.15
```

Comment:
This should't be both in here and in the commands.json file; here is better but you might need to move the `moz*` imports you added into ttthe function where they're actually used.

Reply:
If I remove these requirements from the commands.json file, the code crashes when I try to set up a new venv because mozdownload/install don't get installed there in it.
When I remove them from this file, it could cause the build to fail in Travis if anything calls browse.py because they don't get installed there.

Reply:
So I think this is possible, but it involves ensuring that we install the product-specific requriements later on. Let's do that in a followup.